openeo/rest/connection.py

```diff
@@ -1243,7 +1243,13 @@ def load_collection(
         Load a DataCube by collection id.
 
         :param collection_id: image collection identifier
-        :param spatial_extent: limit data to specified bounding box or polygons
+        :param spatial_extent: limit data to specified bounding box or polygons. Can be provided in different ways:
+            - a shapely geometry
+            - a GeoJSON-style dictionary,
+            - a path (:py:class:`str` or :py:class:`~pathlib.Path`) to a local, client-side GeoJSON file,
+              which will be loaded automatically to get the geometries as GeoJSON construct.
+            - a :py:class:`~openeo.api.process.Parameter` instance.
+            - a bounding box dictionary
```

Comment:
can you put the bounding box dictionary first in the list (that's the most commonly used option, so I would not put  it last)

Reply:
And another thing: also add a `.. versionchanged:: 0.37.0` section at the bottom of this doc block to describe this new feature